src/client/src/App.tsx

```diff
@@ -249,26 +258,33 @@ class App extends React.Component<Props> {
           break;
         case EXTENSION_COMMANDS.RESET_PAGES:
           if (message.payload.resetPages) {
-            this.props.frontendOptions.map((frontend: IOption) => {
-              if (frontend.internalName === message.payload.internalName) {
-                const {
-                  title,
-                  internalName,
-                  version,
-                  author,
-                  licenses
-                } = frontend;
-                this.props.selectFrontend({
-                  title: title as string,
-                  internalName,
-                  version,
-                  author,
-                  licenses
-                });
-              }
-            });
             this.props.resetPageSelection();
-            this.props.history.push(ROUTES.SELECT_PAGES);
+            const { selectedFrontend } = this.props;
+
+            // reset page count
+            const key = `wts.Page.${selectedFrontend.internalName}.Blank`;
+            const PAGE_TYPE_COUNT: IPageCount = {};
+            PAGE_TYPE_COUNT[key] = 1;
+            this.props.updatePageCount(PAGE_TYPE_COUNT);
+
+            // select default blank page
+            const PAGES_SELECTION: ISelected[] = [
+              {
+                title: "Blank",
```

Comment:
Does title need to be internationalized?

Reply:
Hmm, I am not sure. Does the pages that come from coreTS internationalized?

Reply:
The pages response from CoreTS are English only at the moment.

Reply:
Maybe instead of hard code the string, we could make it into a constant?

Reply:
I see, in that case it is not to be internationalized. It does not make sense to make the title a constant here since this is a default configuration that I have put inside of a variable :)